releases.json

```diff
@@ -931,6 +931,14 @@
       "1.7.0-2"
     ]
   },
+  "gtk4": {
+    "dependency_names": [
+      "gtk4"
+    ],
+    "versions": [
+      "4.13.1-1"
```

Comment:
AFAIK, odd numbers are unstable releases, I think we should instead add latest of 4.12.x.

Reply:
Seems to be correct according to [their homepage](https://gtk.org/):
> Latest stable 4.12.3
> Latest unstable 4.13.1
> Latest old stable 3.24.38